docs/source/common/debugging.rst

```diff
@@ -133,43 +207,25 @@ See Also:
     - :func:`~pytorch_lightning.utilities.model_summary.summarize`
     - :class:`~pytorch_lightning.utilities.model_summary.ModelSummary`
 
-----------------
-
-Shorten Epochs
-==============
-
-Sometimes it's helpful to only use a percentage of your training, val, test or predict data (or a set number of batches).
-For example, you can use 20% of the training set and 1% of the validation set.
-
-On larger datasets like Imagenet, this can help you debug or test a few things faster than waiting for a full epoch.
-
-.. testcode::
-
-    # use only 10% of training data and 1% of val data
-    trainer = Trainer(limit_train_batches=0.1, limit_val_batches=0.01)
-
-    # use 10 batches of train and 5 batches of val
-    trainer = Trainer(limit_train_batches=10, limit_val_batches=5)
 
 ----------------
 
-Validation Sanity Check
-=======================
 
-Lightning runs a few steps of validation in the beginning of training.
-This avoids crashing in the validation loop sometime deep into a lengthy training loop.
+*************************************
+Debugging with Distributed Strategies
+*************************************
 
-(See: :paramref:`~pytorch_lightning.trainer.trainer.Trainer.num_sanity_val_steps`
-argument of :class:`~pytorch_lightning.trainer.trainer.Trainer`)
+DDP Debugging
+=============
 
-.. testcode::
+If you are having a hard time debugging DDP on your remote machinem you can debug DDP locally on the CPU:
 
-    # DEFAULT
-    trainer = Trainer(num_sanity_val_steps=2)
+.. code-block:: python
 
-----------------
+    trainer = Trainer(accelerator="cpu", strategy="ddp", devices=2)
 
-Profiling
-=========
+When everything works, switch back to GPU by changing only the accelerator. Note that this will not provide any speed benefits.
```

Comment:
```suggestion
When everything works, switch back to GPU by changing only the accelerator.
```
This cannot be mentioned here. "This" refers to the previous sentence suggesting to switch back to GPU, and this indeed should generally provide speed benefits!!!

Reply:
yeah, it was just referring to the overall trick. But yeah, might be confusing. Updated!